lib/improver/tests/utilities/test_spatial.py

```diff
@@ -286,15 +320,11 @@ def test_not_equal_areas(self):
         """
         Check it raises an error when the input is not an equal areas grid.
         """
-
         self.cube.remove_coord("projection_x_coordinate")
         self.cube.add_dim_coord(
             DimCoord(np.linspace(200.0, 600.0, 3),
                      'projection_x_coordinate', units='m'), 0)
-        with self.assertRaisesRegex(
-                ValueError,
-                "The size of the intervals along the x and y axis"
-                " should be equal."):
+        with self.assertRaises(ValueError):
```

Comment:
This test is for the check_if_grid_is_equal_area function and is a repeat of one below. It can be removed.